src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -83,36 +101,87 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   // Send the request's contents to Datastore in the form of a new ScheduledInterview object.
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String requestedInterviewerId = request.getParameter("interviewer");
-    String requestedIntervieweeId = request.getParameter("interviewee");
-    String userEmail = UserServiceFactory.getUserService().getCurrentUser().getEmail();
-    String userId = UserServiceFactory.getUserService().getCurrentUser().getUserId();
+    String intervieweeEmail = userService.getCurrentUser().getEmail();
+    String intervieweeId = userService.getCurrentUser().getUserId();
+
     // Since UserId does not have a valid Mock, if the id is null (as when testing), it will be
```

Comment:
Mentioned that this comment is not correct above, can you fix the references to this in this file too to match?

Reply:
Handled.